diff -uNr E-MODULES-EXTRA.old/everything-pidgin/src/e_mod_main.c E-MODULES-EXTRA/everything-pidgin/src/e_mod_main.c
--- E-MODULES-EXTRA.old/everything-pidgin/src/e_mod_main.c	2011-08-24 18:04:03.000000000 +0000
+++ E-MODULES-EXTRA/everything-pidgin/src/e_mod_main.c	2011-08-24 23:32:18.696000105 +0000
@@ -617,18 +617,18 @@
 
    buddy_icon_default = eina_stringshare_add("emblem-people");
 
-   plugin_base = EVRY_PLUGIN_BASE("Pidgin", NULL, PIDGIN_CONTACT,
+   plugin_base = EVRY_PLUGIN_BASE(N_("Pidgin"), NULL, PIDGIN_CONTACT,
 				  _begin, _cleanup, _fetch);
    
    evry->plugin_register(plugin_base, EVRY_PLUGIN_SUBJECT, 1);
 
-   act = EVRY_ACTION_NEW("Chat", PIDGIN_CONTACT, 0, "go-next",
+   act = EVRY_ACTION_NEW(N_("Chat"), PIDGIN_CONTACT, 0, "go-next",
 			 _action_chat, NULL);
 
-   act2 = EVRY_ACTION_NEW("Send File", PIDGIN_CONTACT, EVRY_TYPE_FILE, NULL,
+   act2 = EVRY_ACTION_NEW(N_("Send File"), PIDGIN_CONTACT, EVRY_TYPE_FILE, NULL,
 			  _action_send, NULL);
 
-   act3 = EVRY_ACTION_NEW("Write Message", PIDGIN_CONTACT, EVRY_TYPE_TEXT, "go-next",
+   act3 = EVRY_ACTION_NEW(N_("Write Message"), PIDGIN_CONTACT, EVRY_TYPE_TEXT, "go-next",
 			  _action_chat, NULL);
 
    evry->action_register(act, 0);
